frontend/index.html

```diff
@@ -25,13 +29,14 @@
     <style>
         @import url("css/main.css");
         @import url("https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css");
-        @import url('https://fonts.googleapis.com/css?family=Indie+Flower|Flavors|Patrick+Hand|Pacifico&display=swap');
+        @import url('https://fonts.googleapis.com/css?family=Indie+Flower|Flavors|Archivo+Narrow|Patrick+Hand|Pacifico&display=swap');
         @import url("https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css");
     </style>
+  
 </head>
 
-<body>
-    <div id="page-container">
+<body ng-controller="homeCtrl">
+    <div id="page-container" ng-class="{'body-bg': isCurrentRoute('/movies')}">
```

Comment:
I don't think we need to add controllere and logic just to change the background, we can try to do this from the movies view

Reply:
i need background on the body, i don't know how i can do it from movies ..

Reply:
It's ok like this in the end, the homeCtrl should be here and not on the navbar.html which is a child of the parent template